CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Please add to the relevant subsections under Unreleased below on every PR where
 ### Changed
 
 - Updated minimum version of `sentry-sdk` to 2.0.0 to address deprecation warnings. (@jacobromero in https://github.com/wandb/wandb/compare/WB-20890)
+- Added internal method, api._artifact(), to fetch artifacts so that usage events are not created if not called by an external user. (@ibindlish in https://github.com/wandb/wandb/pull/8674)
```

Comment:
could you move it to the unreleased section plz :)

Reply:
Oops. sorry, my bad - totally missed that